src/reuse/spdx.py

```diff
@@ -28,9 +29,19 @@ def run(args, project: Project, out=sys.stdout) -> int:
     with contextlib.ExitStack() as stack:
         if args.file:
             out = stack.enter_context(args.file.open("w", encoding="utf-8"))
-            if args.file.suffix != ".spdx":
+            spdxfile = [
+                pattern
+                for pattern in _IGNORE_SPDX_PATTERNS
+                if pattern.match(args.file.name)
+            ]
+            if not spdxfile:
```

Comment:
```suggestion
            if not any(
                pattern.match(args.file.name)
                for pattern in _IGNORE_SPDX_PATTERNS 
            ):
```

Reply:
Elegant! Implemented in baa1f2f27bb0d0b9b24676fa12d2209513b70567